packages/core/src/lib/ui_builder.js

```diff
@@ -140,7 +140,9 @@ const ui_builder = function () {
         patternName: _.startCase(
           isSubgroupPattern ? pattern.patternSubgroup : pattern.patternGroup
         ),
-        patternDesc: '',
+        patternDesc: isSubgroupPattern
+          ? pattern.patternSubgroupData.markdown
+          : pattern.patternGroupData.markdown,
```

Comment:
Was it really that simple? I know I removed a very complicated functionality from the code in the past, and here it looks easy 👍

Reply:
Well, I even also wondered about that … ;-)

The previous `patternDesc: ''` on line 143 was mainly leading to that always empty pattern description that was introduced by the code provided with commit ede5839457cb630fd164742613eb7f74750f930a – I think that this original place was never really enhanced to a regular implementation, but mainly overlayed by the code parts that you previously removed. Nevertheless the code in `ui_builder.js` isn't just a skeleton but contains some more direct definitions like the `patternName`, so I think that its a perfect place to reintroduce it over here.